pyugrid/read_netcdf.py

```diff
@@ -200,7 +202,7 @@ def load_grid_from_nc_dataset(nc, grid, mesh_name=None, load_data=True):
             array = var[:, :]
             # Fortran order, instead of C order, transpose the array
             # logic below will fail for 3 node or two edge grids.
-            if array.shape[0] == defs['num_ind']:
+            if grid.check_array_order and array.shape[0] == defs['num_ind']:
```

Comment:
One of the test files had three faces with shape (3, 15ish). The code was transposing the node indices array causing all sorts of pain. The attribute ``check_array_order`` was added to ``UGrid``.